### PR TITLE
ita: updated trusted_certs_paths property documentation

### PR DESCRIPTION
**Pull request by pawelpros**

### CODE REVIEW

File: kbs/docs/config.md [hunk 1]
```diff
@@ -37,10 +37,11 @@ The following properties can be set under the `attestation_token_config` section
...
-|----------------------------|---------------|-----------------------------------------------------|----------|-----------|
-| `attestation_token_config` | String        | Attestation token broker type. Valid values: `CoCo` | Yes      | -         |
-| `trusted_certs_paths`        | String Array  | Trusted root certificates file paths (PEM format).  | No       | -         |
+| Property                   | Type         | Description                                                                                                                     | Required | Default |
+|----------------------------|--------------|---------------------------------------------------------------------------------------------------------------------------------|----------|---------|
+| `attestation_token_config` | String       | Attestation token broker type. Valid values: `CoCo`                                                                             | Yes      | -       |
+| `trusted_certs_paths`      | String Array | Trusted Certificates file (PEM format) or a valid Url (file:// or https://) pointing to a JWKSet certificates (local or OpenID) | No       | -       |
```
Comment (fitzthum) [inline kbs/docs/config.md:43]: nit: can you put \` around `file://` and `https://`

**Review by fitzthum — COMMENTED** [review]
One comment. Also would be good to add a slightly longer commit message even though the PR is very simple.

**Comment by pawelpros** [comment]
Commit message updated and comment solved

**Review by fitzthum — APPROVED** [review]
LGTM

**Comment by mythi** [comment]
apologies, I did not do any reviews because of #514. It might take some time to get those changes in so getting this merged makes sense.

What's missing is that for "Valid values" we also have `Jwk`.

**Pull request merged**